server/test/payments.spec.js

```diff
@@ -0,0 +1,49 @@
+require('dotenv').config();
+const chai = require("chai");
+const chaiHttp = require("chai-http");
+const faker = require('faker');
+const app = require("../app.js");
+
+chai.should();
+chai.use(chaiHttp);
+
+describe("/POST Invest", () => {
+    let userInfo, projectId;
+    before((done) => {
+        chai.request(app)
+            .post(`/api/auth/register`)
+            .send({ email: faker.internet.email(), name: faker.name.findName(), password: faker.internet.password() })
+            .end((err, res) => {
+                userInfo = res.body;
+                chai.request(app)
+                    .post(`/api/projects`)
+                    .set({ Authorization: `Bearer ${userInfo.token}` })
+                    .send({
+                        "title": faker.lorem.word(),
+                        "description": faker.lorem.words(),
+                        "industry": faker.random.word(),
+                        "location": faker.address.state(),
+                        "fundingGoal": faker.random.number()
```

Comment:
Is there reason why using double quotes here? and even double-quotes where we use single elsewhere